### PR TITLE
[WIP] harden script with shellcheck

### PR DESCRIPTION
**Pull request by ipa-mdl**
runs shellsheck on all jobs

in progress: fixing all errors

### CODE REVIEW

**Comment by ipa-mdl** [comment]
superseded by #361

File: industrial_ci/_wrap_test.sh [hunk 1]
```diff
@@ -19,9 +19,15 @@
 
 set -e # exit script on errors
 
+# run shellcheck for all files
+files=($(find . -name '*.sh'))
+echo "shellcheck ${files[*]}"
+shellcheck -sbash "${files[@]}"
```
Comment (130s) [inline industrial_ci/_wrap_test.sh]: I might have missed discussion but you seem to have dropped running `shellcheck` in ICI in the subsequent PRs. Just curious what the decision was?

Comment (130s) [reply industrial_ci/_wrap_test.sh]: I figured it's [run in .travis.yml](https://github.com/ros-industrial/industrial_ci/pull/369/files#diff-354f30a63fb0907d4ad57269548329e3R84).

Comment (ipa-mdl) [reply industrial_ci/_wrap_test.sh]: It got merged in #369, which you approved as well ;)

**Pull request closed without merging**